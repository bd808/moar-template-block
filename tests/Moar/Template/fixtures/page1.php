<?php
  use Moar\Template\Block as T;
  include dirname(__FILE__) . '/base.php';

  T::block('content', 'trim');
?>
Page 1 content.
<?php
  T::endblock('content');

  T::block('title', array('trim'));
?>
PAGE 1
<?php
  T::endblock('title');
