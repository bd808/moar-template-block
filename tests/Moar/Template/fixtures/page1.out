<!DOCTYPE HTML>
<html>
<head>
<title>PAGE 1 - Tests</title>
</head>
<body>
<div id="content">Page 1 content.</div>
</body>
</html>
